chcemvediet/apps/inforequests/tests/__init__.py

```diff
@@ -90,7 +90,7 @@ def _post_teardown(self):
     def _create_inforequest_scenario__action(self, inforequest, branch, args):
         action_name = args.pop(0)
         action_type = getattr(Action.TYPES, action_name.upper())
-        action_extra = args.pop() if args and isinstance(args[0], dict) else {}
+        action_extra = args.pop(0) if args and isinstance(args[0], dict) else {}
```

Comment:
Asi si to prehliadol pri pisani, doteraz sa to nepouzivalo, resp. vzdy v tvare `(u'disclosure', dict(disclosure_level=Action.DISCLOSURE.PARTIAL))`, teda iba s 2 argumentami a `.pop()` a `.pop(0)` sa spravali rovnako,  vybrali posledny, jediny zvysny argument

Reply:
Good catch. Vdaka.